finishline

```diff
@@ -466,7 +469,9 @@ if __name__ == '__main__':
     # dynamically build the arguments to pass into jira.client.JIRA
     client_kwargs = {
         'server': args.server,
-        'options': dict(verify=args.cacert),
+        'options': dict(verify=args.cacert, auth_url=args.auth_path),
+        'validate': True,
```

Comment:
In addition to optionally setting the auth_url as ashcrow mentioned above, Validate should only be set when the auth-path gets specified.

Reply:
Hmm... @accorvin you sure? The [api docs](https://jira.readthedocs.io/en/master/api.html#jira) mention:

> validate – If true it will validate your credentials first. Remember that if you are accesing JIRA as anononymous it will fail to instanciate.

Since the current code does not allow any anonymous communication with JIRA (falls back to kerberos) I thought, validating is always a benefit to know that your credentials are correct. That certainly goes for basic auth. Yet now I'm wondering if it makes sense for Kerberos at all ...

Reply:
I'm not sure about the falling back to kerberos thing, but more often than not I run finishline without supplying any credentials and without having a valid kerberos ticket, so I generally run this as anonymous. In my testing, this failed when validate was set to true.

Reply:
@accorvin aaah I see. No worries. Will change.